src/components/existing-request/existing-request-display.vue

```diff
@@ -111,19 +111,7 @@
             <!-- action buttons -->
             <v-col cols="3" class="py-0">
               <v-row dense>
-                <template v-if="pendingPayment">
```

Comment:
The UI should not decide what actions are displayed.

The API is already specifying the correct actions.

The reason duplicate actions were not displayed is because of the `v-else` on line 126.